src/scripts/components/stories/video-js/video-js.tsx

```diff
@@ -90,6 +92,8 @@ const VideoJS: FunctionComponent<Props> = ({
         ref={videoRef}
         className="video-js vjs-big-play-centered"
         style={{height: '100%'}}
+        // @ts-ignore
+        onPlay={event => onPlay(event.target)}
```

Comment:
I guess this ts-ignore could be avoided like this? Or what is the issue here?
```suggestion
        onPlay={event => onPlay(event.target as VideoJsPlayer)}
```

Reply:
Yep, i also tried this one and EventTarget as well. 😅 
`Conversion of type 'EventTarget' to type 'VideoJsPlayer' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.`

Reply:
better than the ignore I would say. This way we atleast can see what is being done and what the issue was